src/components/UserRepos/UserRepos.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import { Repo } from "../../interface";
+import Button from "../Button/Button";
+import ReposDetails from "../ReposDetails/ReposDetails";
+
+interface repoProps {
```

Comment:
remember, types (interface, types, classes) always start with an uppercase.
```suggestion
interface RepoProps {
```